src/pages/api/mailer.tsx

```diff
@@ -0,0 +1,45 @@
+"use strict";
+import nodemailer from "nodemailer";
+import type { NextApiRequest, NextApiResponse } from "next";
+
+export default async function (req: NextApiRequest, res: NextApiResponse) {
+  try {
+    const transporter = nodemailer.createTransport({
+      host: "smtp.gmail.com",
+      port: 587,
+      secure: false, // true for 465, false for other ports
+      auth: {
+        user: process.env.EMAIL_API_EMAIL,
+        pass: process.env.EMAIL_API_PASSWORD
+      }
+    });
+
+    const textString =
+      "New contact form request from MathSoc website:" +
+      "\n" +
+      "First Name: " +
```

Comment:
Use template literals.
Also, you can write something more natural like `Message from ${firstName} ${lastName} <${email}> via MathSoc website: ` as opposed to "New contact form request..." etc..